app/controllers/ansible_roles_controller.rb

```diff
@@ -6,18 +6,9 @@ class AnsibleRolesController < ::ApplicationController
   include ForemanAnsible::Concerns::ImportControllerHelper
   include ::ForemanAnsible::AnsibleRolesDataPreparations
   def index
-    @ansible_roles = resource_base.search_for(params[:search],
-                                              :order => params[:order]).
-                     paginate(:page => params[:page],
-                              :per_page => params[:per_page])
   end
 
   def destroy
```

Comment:
If this is a dummy, should the whole method remain?

Reply:
Removed the empty methods.